CLAUDE.md

```
@@ -100,6 +104,6 @@ All HTTP requests go through the centralized `WandererNotifier.Http` module whic
 
 ### Feature Flags
 Features can be toggled via environment variables ending in `_ENABLED`:
-- `KILLMAIL_NOTIFICATION_ENABLED`
-- `SYSTEM_NOTIFICATION_ENABLED`
-- `CHARACTER_NOTIFICATION_ENABLED`
\ No newline at end of file
+- `KILL_NOTIFICATIONS_ENABLED` - Enable/disable kill notifications
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)